SashaZaitsev/HW5/05. Loops.py

```diff
@@ -0,0 +1,26 @@
+# Программа, що перетворює список з класу int на float
+numbers = [1, 2 , 3, 4, 5]
+print("Перетворюємо числа у тип float")
+
+for i in numbers:
+    i = float(i)
+    print(i)
+
+# Програма виводе ряд Фібоначі до числа n
+f1 = 1
+f2 = 1
+
+n = int(input("Введіть число до ряду Фібоначчі: "))
+
+while n > 2:
+    f1, f2 = f2, f1 + f2
+    print(f2)
+    n -= 1
+
+# Программа виводе факторіал числа х
+x = int(input("Введіть число, факторіал якого хочете знайти: "))
```

Comment:
look at this version of code
input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)